modules/mono/editor/GodotTools/GodotTools/Ides/GodotIdeManager.cs

```diff
@@ -59,7 +59,7 @@ private string GetExternalEditorIdentity(ExternalEditorId editorId)
             switch (editorId)
             {
                 case ExternalEditorId.None:
-                    return null;
+                    return string.Empty;
```

Comment:
Converting to `string.Empty` felt more appropriate for the enum-to-string usecase, especially when it would eventually need to be converted regardless to work as a dictionary key